deploy/docker-only/docker-compose.yml

```diff
@@ -21,6 +21,16 @@ services:
     restart: always
     environment:
       - reschedule:on-node-failure
+  catalogue-db:
+    image: mysql
+    hostname: catalogue-db
+    restart: always
+    volumes:
+      - ./data:/docker-entrypoint-initdb.d
```

Comment:
So if I'm right, mysql will automagically read from the data/dump.sql file and populate the DB.

Is there any way of doing this externally, to run the sql commands against the db so that this insertion of data will be easier to implement for other deploys? E.g. we don't really want to mount on K8s, because then we'd have to write code to download the sql file to the right place.